### PR TITLE
EPSS

### PR DESCRIPTION
**Pull request by tschmidtb51**
- addresses parts of https://github.com/oasis-tcs/csaf/issues/622
- add EPSS to schema
- adapt prose
- add paragraphs about EPSS as last section in content to avoid confusion
- add new key to guidance on size- add mandatory test 6.1.51 to test for inconsistent timestamps
- add optional test for EPSS older than `current_release_date`
- add invalid examples
- add vaild examples

### CODE REVIEW

File: csaf_2.1/json_schema/csaf_json_schema.json [hunk 1]
```diff
@@ -1288,6 +1288,36 @@
...
+                          "title": "Percentile",
+                          "description": "Contains the rank ordering of probabilities from highest to lowest.",
+                          "type": "string",
+                          "pattern": "^(([0]\\.([0-9])+)|([1]\\.[0]+))$"
+                        },
+                        "probability": {
+                          "title": "Probability",
+                          "description": "Contains the likelihood that any exploitation activity for this Vulnerability is being observed in the 30 days following the given timestamp.",
+                          "type": "string",
+                          "pattern": "^(([0]\\.([0-9])+)|([1]\\.[0]+))$"
```
Comment (sthagen) [inline csaf_2.1/json_schema/csaf_json_schema.json:1311]: I wonder if we can declare patterns upfront and reuse them.

File: csaf_2.1/json_schema/csaf_json_schema.json [hunk 2]
```diff
@@ -1288,6 +1288,36 @@
...
+                        "percentile",
+                        "probability",
+                        "timestamp"
+                      ],
+                      "properties": {
+                        "percentile": {
+                          "title": "Percentile",
+                          "description": "Contains the rank ordering of probabilities from highest to lowest.",
+                          "type": "string",
+                          "pattern": "^(([0]\\.([0-9])+)|([1]\\.[0]+))$"
```
Comment (sthagen) [inline csaf_2.1/json_schema/csaf_json_schema.json:1305]: Maybe we are better off using a number here and adding test for that number to be in the expected range and display form?

File: csaf_2.1/prose/edit/src/tests-01-mndtr-51-inconsistent-epss-timestamp.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+`revision_history` if the document status is `final` or `interim`.
+As the timestamps might use different timezones, the sorting MUST take timezones into account.
+
+The relevant path for this test is:
+
+```
+    /vulnerabilities[]/metrics[]/content/epss/timestamp
+```
+
+*Example 1 (which fails the test):*
```
Comment (sthagen) [inline csaf_2.1/prose/edit/src/tests-01-mndtr-51-inconsistent-epss-timestamp.md:13]: I can add the corresponding example entry after merge into the revision branch.

File: csaf_2.1/prose/edit/src/tests-02-opt-41-old-epss-timestamp.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+`revision_history` if the document status is `final` or `interim`.
+As the timestamps might use different timezones, the sorting MUST take timezones into account.
+
+The relevant path for this test is:
+
+```
+    /vulnerabilities[]/metrics[]/content/epss/timestamp
+```
+
+*Example 1 (which fails the test):*
```
Comment (sthagen) [inline csaf_2.1/prose/edit/src/tests-02-opt-41-old-epss-timestamp.md:13]: I can add the corresponding example entry after merge into the revision branch.

File: csaf_2.1/prose/edit/src/tests-02-opt-41-old-epss-timestamp.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
+### Old EPSS Timestamp
```
Comment (sthagen) [inline csaf_2.1/prose/edit/src/tests-02-opt-41-old-epss-timestamp.md:1]: I can add the corresponding section label entries after merge into the revision branch.

**Review by sthagen — APPROVED** [review]
LGTM.

File: csaf_2.1/prose/edit/src/tests-01-mndtr-51-inconsistent-epss-timestamp.md (hunk 3 above)
Comment (tschmidtb51) [reply csaf_2.1/prose/edit/src/tests-01-mndtr-51-inconsistent-epss-timestamp.md:13]: Thank you.

File: csaf_2.1/prose/edit/src/tests-02-opt-41-old-epss-timestamp.md (hunk 4 above)
Comment (tschmidtb51) [reply csaf_2.1/prose/edit/src/tests-02-opt-41-old-epss-timestamp.md:13]: Thank you.

File: csaf_2.1/prose/edit/src/tests-02-opt-41-old-epss-timestamp.md (hunk 5 above)
Comment (tschmidtb51) [reply csaf_2.1/prose/edit/src/tests-02-opt-41-old-epss-timestamp.md:1]: Thank you.

File: csaf_2.1/json_schema/csaf_json_schema.json (hunk 1 above)
Comment (tschmidtb51) [reply csaf_2.1/json_schema/csaf_json_schema.json:1311]: I though about that last night as well. My conclusion was: For only 2 instances with just patterns, the overhead is too big.

File: csaf_2.1/json_schema/csaf_json_schema.json (hunk 2 above)
Comment (tschmidtb51) [reply csaf_2.1/json_schema/csaf_json_schema.json:1305]: I'm +-0 about that idea. The official API at https://api.first.org/epss/ has the response as string. So it might be the least surprise, if we stick with the strings. However, from the JSON schema point of view, the `number` really makes sense. We wouldn't need to do an extra test - we can use `minimum` and `maximum`.

File: csaf_2.1/prose/edit/src/tests-02-opt-41-old-epss-timestamp.md (hunk 5 above)
Comment (tschmidtb51) [reply csaf_2.1/prose/edit/src/tests-02-opt-41-old-epss-timestamp.md:1]: I just added them in [3b57346](https://github.com/oasis-tcs/csaf/pull/969/commits/3b57346a5c4bd48a511058de0c3b9cd2371a04e7) as well as put that into the bind.txt.
